frontend/public/components/chargeback.tsx

```diff
@@ -371,6 +372,7 @@ const ReportsPage_: React.SFC<ReportsPageProps> = props => {
   if (flagPending(props.flags[FLAGS.CHARGEBACK])) {
     return <LoadingBox />;
   }
+  setActiveNamespace('openshift-metering');
```

Comment:
I wouldn't do it this way... Let's just make the left nav link go directly to this NS.

Reply:
ok, switched to nav link (see last commit), however not as restrictive as users can switch project/namespace to something else other than 'openshift-metering'; whereas previous implementation always defaulted to 'openshift-metering' when on ReportsPage